src/ngraph/runtime/backend.hpp

```diff
@@ -109,6 +109,11 @@ class ngraph::runtime::Backend
                                                 ngraph::pass::PassConfig& pass_config,
                                                 bool enable_performance_data = false);
 
+    /// \brief Loads a previously saved Executable object from a stream.
+    /// \param input_stream the opened input stream containing the saved Executable
+    /// \returns A compiled function or nullptr on failure
+    virtual std::shared_ptr<Executable> load(std::istream& input_stream);
```

Comment:
So far nGraph error communication has been through exception - should we stick to that? That way backends can propagate the error messages to the end user (via the framework).